astromon/web/celmon.py

```diff
@@ -28,57 +28,6 @@
 )
 
 
-def good_obs(matches):
```

Comment:
What's happening with this deleted code? Was this orphaned from earlier development?

Reply:
Yes, this was orphaned and the functionality is now in [db.cross_match](https://github.com/sot/astromon/blob/master/astromon/cross_match.py#L457). `db.get_matches` also calls it from there.

Reply:
that filtering was checked in another PR. The change here does nothing.